finagle-netty4/src/test/scala/com/twitter/finagle/netty4/Netty4SslTest.scala

```diff
@@ -0,0 +1,111 @@
+package com.twitter.finagle.netty4
+
+import java.net.{InetAddress, InetSocketAddress, SocketAddress}
+import java.nio.charset.StandardCharsets.UTF_8
+
+import com.twitter.finagle
+import com.twitter.finagle.Stack.Params
+import com.twitter.finagle._
+import com.twitter.finagle.client.{StackClient, StdStackClient, Transporter}
+import com.twitter.finagle.dispatch.{SerialClientDispatcher, SerialServerDispatcher}
+import com.twitter.finagle.param.Label
+import com.twitter.finagle.ssl.client.{SslClientConfiguration, SslClientEngineFactory}
+import com.twitter.finagle.ssl.server.{SslServerConfiguration, SslServerEngineFactory}
+import com.twitter.finagle.ssl.{ClientAuth, Engine}
+import com.twitter.finagle.transport.Transport
+import com.twitter.finagle.transport.Transport.ServerSsl
+import com.twitter.util.{Await, Future}
+import io.netty.channel.ChannelPipeline
+import io.netty.handler.codec.string.{StringDecoder, StringEncoder}
+import io.netty.handler.codec.{DelimiterBasedFrameDecoder, Delimiters}
+import io.netty.handler.ssl.SslContextBuilder
+import io.netty.handler.ssl.util.SelfSignedCertificate
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+import org.scalatest.concurrent.{Eventually, IntegrationPatience}
+import org.scalatest.junit.JUnitRunner
+
+object Netty4SslTest {
+  case class Client(clientCert: SelfSignedCertificate, serverCert: SelfSignedCertificate,
+                    params: Params = Params.empty,
```

Comment:
indentation for classes should be like

```
case class ClassName(
    arg1: Arg1,
    arg2: Arg2)
```

so that each argument is indented by four spaces.